client/i18n/locales/japanese/trending.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
As we expand our Japanese news platform, we'll be able to use Japanese links here as well.